spec/controllers/reorder_timeline_entries_controller_spec.rb

```diff
@@ -26,4 +26,94 @@
       expect(response.status).to eq(403)
     end
   end
+
+  describe "PUT Coronavirus reorder timeline entries page" do
+    let(:first_timeline_entry) { create(:timeline_entry, coronavirus_page: coronavirus_page) }
+    let(:second_timeline_entry) { create(:timeline_entry, coronavirus_page: coronavirus_page) }
+
+    before do
+      setup_github_data
+      stub_coronavirus_publishing_api
+      stub_user.permissions = ["signin", "Coronavirus editor", "Unreleased feature"]
+
+      first_timeline_entry.position = 1
+      first_timeline_entry.save!
+      second_timeline_entry.position = 2
+      second_timeline_entry.save!
```

Comment:
I don't really understand. Can I set the position in the `let!`? As new timeline entries are given position 1, I thought it might not be obvious what the "starting" position for each timeline entry is, so I decided to explicitly set it.